mne_bids/datasets.py

```diff
@@ -40,7 +40,7 @@ def fetch_faces_data(data_path=None, repo='ds000117', subject_ids=[1]):
         if not os.path.exists(data_path):
             os.makedirs(data_path)
 
-    for subject_id in subject_ids:
+    for subject_id in subject_ids:  # pragma: no cover
```

Comment:
so you increase coverage by ignoring parts of the code? What line numbers does this include?

Reply:
the `no cover` covers anything within the `for` loop.
And the point of `pragma: no cover` is to tell the coverage checker to ignore code that is either never meant to be reached or covered by the tests (as is our case), or debugging code.
The point is you either increase the coverage by actually writing a test to cover this code (which involves downloading 7.9Gb of data each time...), or you ignore it as the coverage checker will always say it is never covered.

Reply:
okay fair enough!